proto/ptypes/replica.proto

```diff
@@ -173,3 +176,22 @@ message PrepareRemoveAction {
   string source = 2;
   string target = 3;
 }
+
+message RepProfilerOpCommonRequest {
+    string op = 1;
+}
+
+message RepProfilerOpEnableRequest {
+    string addr = 1;
+    int32 port_number = 2;
+}
```

Comment:
The API definition looks a little bit weird. I prefer to use either 3 separate APIs (enable, disable, show) or one single API (ReplicaProfileRequest --op <enable/disable/show> --port-number <for enable op>; the output is the port number (enable/show) or empty(disable))

Reply:
Thanks for the suggestion!
I combine them into one. `RepProfilerOp` and `RepProfilerOpResponse`.
Please help to recheck it! Thanks!